### PR TITLE
Phases 4 & 5, Integrations and Troubleshooting

### PR DESCRIPTION
**Pull request by PeloWriter**
Adding the final templates, focused on integrations and troubleshooting. Also updated and simplified the template guidelines.

### CODE REVIEW

**Comment by HazAT** [comment]
We should add this to the troubleshooting section for JS
https://github.com/getsentry/sentry-javascript/issues/1003#issuecomment-380310913
ref: https://twitter.com/rektide/status/1283795989409730560

File: src/collections/_documentation/sdks/javascript/index.md [hunk 1]
```diff
@@ -73,22 +73,34 @@ Upload source maps using our Webpack plugin or TypeScript.
...
+# Using yarn
+$ yarn add @sentry/browser @sentry/apm
+
+# Using npm
+$ npm install @sentry/browser @sentry/apm
+```
+
+Alternatively, instead of `npm` packages, you can use our pre-built CDN bundle that combines both @sentry/browser and @sentry/apm:
+
+```bash
```
Comment (HazAT) [inline src/collections/_documentation/sdks/javascript/index.md:88]:
```suggestion
```html
```

File: src/collections/_documentation/sdks/javascript/index.md [hunk 2]
```diff
@@ -73,22 +73,34 @@ Upload source maps using our Webpack plugin or TypeScript.
...
+<script src="https://browser.sentry-cdn.com/5.19.1/bundle.apm.min.js" integrity="sha384-qcuJnBVywcOnVTsYC4W46ENmLFtgTt1rojscXpw6pgE6BDJ+MSj3AgdT897AgysN" crossorigin="anonymous"></script>
```
Comment (HazAT) [inline src/collections/_documentation/sdks/javascript/index.md:89]:
```suggestion
<script src="https://browser.sentry-cdn.com/{% sdk_version sentry.javascript.browser %}/bundle.apm.min.js" integrity="{% sdk_cdn_checksum sentry.javascript.browser latest bundle.min.js %}" crossorigin="anonymous"></script>
```

File: src/collections/_documentation/sdks/javascript/integrations/custom.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: Custom Integrations
+excerpt: ""
+---
+
+{%- capture __custom-integration -%}
+
+```bash
```
Comment (HazAT) [inline src/collections/_documentation/sdks/javascript/integrations/custom.md:8]:
```suggestion
```js
```

File: src/collections/_documentation/sdks/javascript/integrations/default.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+{%- endcapture -%}
+
+
+{%- capture __remove-integration -%}
+
+```js
+import * as Sentry from '@sentry/browser';
+
+Sentry.init({
+  dsn: 'https://bc367fd982144d75923219dca8f35ad1@o1.ingest.sentry.io/2783252',
```
Comment (HazAT) [inline src/collections/_documentation/sdks/javascript/integrations/default.md:99]:
```suggestion
  dsn: '___PUBLIC_DSN___',
```

**Comment by HazAT** [comment]
I hope I caught all `bash` to `-> `js` and `___PUBLIC_DSN___` 😅

File: src/collections/_documentation/sdks/javascript/index.md (hunk 1 above)
Comment (PeloWriter) [reply src/collections/_documentation/sdks/javascript/index.md:88]: Thanks.

File: src/collections/_documentation/sdks/javascript/index.md (hunk 2 above)
Comment (PeloWriter) [reply src/collections/_documentation/sdks/javascript/index.md:89]: Thanks.

File: src/collections/_documentation/sdks/javascript/integrations/default.md (hunk 4 above)
Comment (PeloWriter) [reply src/collections/_documentation/sdks/javascript/integrations/default.md:99]: oof. Thanks for the catch.

File: src/collections/_documentation/sdks/javascript/integrations/custom.md (hunk 3 above)
Comment (PeloWriter) [reply src/collections/_documentation/sdks/javascript/integrations/custom.md:8]: Thanks.

File: src/collections/_documentation/sdks/javascript/troubleshooting/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+Most community CDNs properly set an `Access-Control-Allow-Origin` header.
+
+```bash
+ $ curl --head https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.js | \
+ grep -i "access-control-allow-origin"
+
+ Access-Control-Allow-Origin: *
+ ```
+
+## Missing Console Log Line Numbers for Breadcrumbs
```
Comment (HazAT) [inline src/collections/_documentation/sdks/javascript/troubleshooting/troubleshooting.md:44]:
```suggestion
## instrument.js Line Numbers for Console Log statements
```

File: src/collections/_documentation/sdks/javascript/troubleshooting/troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+If you are using our SDK version earlier than Version 3, you need to add `/@sentry/` instead of `/raven-js/` to ensure breadcrumbs capture console log line numbers.
```
Comment (HazAT) [inline src/collections/_documentation/sdks/javascript/troubleshooting/troubleshooting.md:46]:
```suggestion
If you are seeing `instrument.js` in you console while debugging, add Sentry to your framework [blackboxing](https://developer.chrome.com/devtools/docs/blackboxing) settings like: `/@sentry/` to tell Chrome to ignore the SDK stackframes when debugging. 
```

File: src/collections/_documentation/sdks/javascript/troubleshooting/troubleshooting.md (hunk 5 above)
Comment (PeloWriter) [reply src/collections/_documentation/sdks/javascript/troubleshooting/troubleshooting.md:44]: Thanks.

File: src/collections/_documentation/sdks/javascript/troubleshooting/troubleshooting.md (hunk 6 above)
Comment (PeloWriter) [reply src/collections/_documentation/sdks/javascript/troubleshooting/troubleshooting.md:46]: Thanks. Reworded slightly:

If `instrument.js` displays in your console while debugging, add Sentry to your framework [blackboxing](https://developer.chrome.com/devtools/docs/blackboxing) settings like: `/@sentry/` so Chrome ignores the SDK stackframes when debugging.

**Pull request merged**